CRM/CiviAwards/Enable/ActivateAwardsMenus.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use CRM_CiviAwards_Service_ApplicantManagementMenu as ApplicantManagementMenu;
+use CRM_CiviAwards_Helper_CaseTypeCategory as CaseTypeCategoryHelper;
+
+/**
+ * Activates the menu items for Civi Awards.
+ */
+class CRM_CiviAwards_Enable_ActivateAwardsMenus {
+
+  /**
+   * Activates the Awards menu items.
+   */
+  public function apply() {
+    $applicationMenu = new ApplicantManagementMenu();
+    $applicationMenu->activateItem(CaseTypeCategoryHelper::AWARDS_CASE_TYPE_CATEGORY_NAME);
+  }
```

Comment:
We need to disable not only award menus but also menus for case categories belonging to applicant management.
The following slightly modified code should do just fine. Same loops need to be in function to disable too.
```php

  public function apply() {
    $caseCategoryDetails = array_column(CRM_Civicase_Helper_CaseCategory::getCaseCategories(), 'name', 'value');
    $applicantCaseCategoryIds = CRM_CiviAwards_Helper_CaseTypeCategory::getApplicantManagementCaseCategories();
    $applicationMenu = new ApplicantManagementMenu();
    foreach ($applicantCaseCategoryIds as $caseCategoryValue) {
        $applicationMenu->activateItem($caseCategoryDetails[$caseCategoryValue]);
    }
  }
```